astropy/modeling/core.py

```diff
@@ -2657,7 +2658,7 @@ def __call__(self, *args, **kw):
         if with_bbox and bbox is not None:
             # first check inputs are consistent in shape
             input_shape = _validate_input_shapes(args, (), self._n_models,
-                                                 self.model_set_axis, self.standard_broadcasting)
+                                                 self.model_set_axis)
```

Comment:
Will this cause problems if self.standard_broadcasting is False (well, it looks like this doesn't change past behavior, but still, is it not a problem?)